### DISCUSSION TITLE
Meta-generation failed

### DISCUSSION THREAD

**Discussion by Abhishek2408** [coding-geek21/classDeck#50] [General]
Can anyone please help me to solve this issue? I'm getting this error while doing setup
![image](https://user-images.githubusercontent.com/87486132/156896363-673a85e2-d10b-4bf8-9653-d500b5aac10d.png)

**Comment by HarshKumarChoudary** [comment]
I too faced this one. This means you have to install all the packages one by one.

**Comment by Abhishek2408** [comment]
@HarshKumarChoudary I tried but i think now it is showing python version issue.

**Comment by yaswanthsaivendra** [comment]
What version of python u r using?? and what's your system os??

**Comment by Abhishek2408** [comment]
@yaswanthsaivendra i have solve that issue. but on my side student signup is not working!!!

**Comment by yaswanthsaivendra** [comment]
@Abhishek2408 , refer https://github.com/coding-geek21/classDeck/issues/23#issuecomment-1057902257

**Comment by Abhishek2408** [comment]
@yaswanthsaivendra ya got it. Thanks
